torch/csrc/api/src/nn/modules/activation.cpp

```diff
@@ -189,5 +189,17 @@ void CELUImpl::pretty_print(std::ostream& stream) const {
   stream << ")";
 }
 
+// ============================================================================
+
+Tensor SigmoidImpl::forward(const Tensor& input) {
+  return sigmoid(input);
```

Comment:
nit: would be great to write:
```suggestion
  return torch::sigmoid(input);
```
to strictly match Python version :D